app/src/main/java/org/oppia/android/app/policies/PoliciesActivityPresenter.kt

```diff
@@ -0,0 +1,57 @@
+package org.oppia.android.app.policies
+
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import org.oppia.android.R
+import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.app.translation.AppLanguageResourceHandler
+import javax.inject.Inject
+
+/** The presenter for [PoliciesActivity]. */
+@ActivityScope
+class PoliciesActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val resourceHandler: AppLanguageResourceHandler
+) {
+
+  private lateinit var toolbar: Toolbar
+
+  /** Handles onCreate() method of the [PoliciesActivity]. */
+  fun handleOnCreate(policies: Int) {
+    activity.setContentView(R.layout.policies_activity)
+    setUpToolbar(policies)
+    activity.supportActionBar!!.setDisplayHomeAsUpEnabled(true)
+
+    toolbar.setNavigationOnClickListener {
+      activity.finish()
+    }
+
+    if (getPoliciesFragment() == null) {
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.policies_fragment_placeholder,
+        PoliciesFragment.newInstance(policies)
+      ).commitNow()
+    }
+  }
+
+  private fun setUpToolbar(policies: Int) {
+    toolbar = activity.findViewById<View>(R.id.policies_activity_toolbar) as Toolbar
+
+    when (policies) {
+      Policies.PRIVACY_POLICY.ordinal ->
+        toolbar.title = resourceHandler.getStringInLocale(R.string.privacy_policy_title)
+      Policies.TERMS_OF_SERVICE.ordinal ->
+        toolbar.title = resourceHandler.getStringInLocale(R.string.terms_of_service_title)
+    }
```

Comment:
```suggestion
    toolbar.title = when (policies) {
      Policies.PRIVACY_POLICY.ordinal ->
        resourceHandler.getStringInLocale(R.string.privacy_policy_title)
      Policies.TERMS_OF_SERVICE.ordinal ->
        resourceHandler.getStringInLocale(R.string.terms_of_service_title)
    }
```

Prefer this being an exhaustive when (which will be possible if we retain type safety on the enum) since then new additions to the enum will create a compiler error here rather than a bug.